<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T26Dialog</class>
 <widget class="QDialog" name="T26Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>441</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>Lock and Fuse Bit Edit ATTiny26</string>
   </property>
  </widget>
  <widget class="QFrame" name="highFuseFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>351</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QLabel" name="highFuseLabel_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>High Fuse Bits</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="resetDisableBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>37</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Disables the external reset pin for use as a general port</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>External Reset Disable  </string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="enableSerialBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Enables the use of serial (SPI) programming of the target</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Enable Serial Programming</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="preserveEEPROMBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Keeps the EEPROM contents following a chip erase action</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Preserve EEPROM on reset</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="brownoutBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>121</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Disabled</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.7V</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4.0V</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="brownoutLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Brownout Level</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="fuseFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>351</width>
     <height>141</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QCheckBox" name="clockOptBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Output the clock signal on pin PB0</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Clock Option (see datasheet)</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="clockPLLBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Divide down the clock frequency by 8</string>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Use PLL for Internal Clock</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fuseLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Fuse Bits</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="startupTimeBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="startupTimeBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="clockSourceBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="clockSourceBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="clockSourceBox_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="clockSourceBox_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="clockSourceLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Clock Source Selection.. Refer to the datasheets. The settings are not inverted and match those of the datasheet.</string>
    </property>
    <property name="text">
     <string>Clock Source Selection</string>
    </property>
   </widget>
   <widget class="QLabel" name="startupTimeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>This depends on the choice of clock source. Refer to the datasheets. The settings are not inverted and match those of the datasheet.</string>
    </property>
    <property name="text">
     <string>Startup Time Selection</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="lockFrame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <widget class="QLabel" name="memoryLockLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory Lock</string>
    </property>
   </widget>
   <widget class="QComboBox" name="memoryLockBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>36</y>
      <width>191</width>
      <height>27</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Selects the type of locking of memory, either none, read only or read/write. Fuse bits are locked in the latter two modes.</string>
    </property>
    <item>
     <property name="text">
      <string>No Memory Lock</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Memory/Fuse Write Lock</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Memory/Fuse/BootLock Read and Write Lock</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lockLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string>Lock Bits</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="lockWriteButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Write Locks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="highFuseWriteButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Write HFBs</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fuseWriteButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Write FBs</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>100</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lock bits are
cleared by a Chip
Erase (but not in
a bootloader).</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>370</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Startup time and
clock source bits
are as shown in
the datasheet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>440</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="warningLabel_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>371</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Labelled checkbox settings are the inverse of those given in the datasheet
except for the Startup Time and Clock Source check bits.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
